registration/include/pcl/registration/impl/ndt.hpp

```diff
@@ -95,26 +95,23 @@ NormalDistributionsTransform<PointSource, PointTarget>::computeTransformation (P
   }
 
   // Initialize Point Gradient and Hessian
-  point_gradient_.setZero ();
-  point_gradient_.block<3, 3>(0, 0).setIdentity ();
+  point_jacobian_.setZero ();
+  point_jacobian_.block<3, 3>(0, 0).setIdentity ();
   point_hessian_.setZero ();
 
   Eigen::Transform<float, 3, Eigen::Affine, Eigen::ColMajor> eig_transformation;
   eig_transformation.matrix () = final_transformation_;
 
   // Convert initial guess matrix to 6 element transformation vector
-  Eigen::Matrix<double, 6, 1> p, delta_p, score_gradient;
+  Eigen::Matrix<double, 6, 1> transform, score_gradient;
   Eigen::Vector3f init_translation = eig_transformation.translation ();
   Eigen::Vector3f init_rotation = eig_transformation.rotation ().eulerAngles (0, 1, 2);
-  p << init_translation (0), init_translation (1), init_translation (2),
-  init_rotation (0), init_rotation (1), init_rotation (2);
+  transform << init_translation.cast<double>(), init_rotation.cast<double>();
```

Comment:
You mentioned elsewhere that `<<` is sometimes slower. Is this usage also impacting performance?

Reply:
I think it doesn't matter because it's called only once per registration.

BTW, ```<<``` may not get slower with recent Eigen and compiler (I observed the performance issues three years ago), but I'm not so sure.